tests/testthat/test-translate.r

```diff
@@ -146,6 +146,22 @@ test_that("str_trim() translates correctly ", {
       sql("LTRIM(RTRIM(\"field_name\"))"))
 })
 
+# lubridate -------------------------------------------
 
+test_that("as_date() translates correctly ", {
+  expect_equivalent(
```

Comment:
These tests should be organised to fit on a single line (or as close as possible)